src/components/ebay-carousel/style.less

```diff
@@ -67,6 +69,35 @@
         }
     }
 
+    &__play, &__pause {
+        z-index: 2;
+        bottom: 16px; // accounts for padding.
+        right: 0;
+        margin-right: 19px;
+        margin-bottom: 19px;
+        position: absolute;
+        background: 0;
+        border: 0;
+        padding: 0;
+        background-color: rgba(0,0,0,0.45);
+        height: 40px;
+        width: 40px;
+        border-radius: 50%;
+        text-align: center;
+        color: @ds6-color-g201-gray;
+        box-sizing: border-box;
+
+        &&& svg {
```

Comment:
I think you were aiming for 3 play and 3 pause, but it ends up like 
```css
.carousel__play.carousel__play.carousel__play svg, 
.carousel__play.carousel__play.carousel__pause svg, 
.carousel__play.carousel__pause.carousel__play svg, 
.carousel__play.carousel__pause.carousel__pause svg, 
.carousel__pause.carousel__play.carousel__play svg, 
.carousel__pause.carousel__play.carousel__pause svg, 
.carousel__pause.carousel__pause.carousel__play svg,
.carousel__pause.carousel__pause.carousel__pause svg
```

Reply:
The more `&`'s the more frustrated I am with css. (Will try to clean this up)